lib/2_uppercases_all_letters.php

```diff
@@ -7,9 +7,29 @@
 // Run `bin/check` from the command line to execute the automated tests.
 
 function uppercasesAllLetters($str) {
-  // Your code goes here.
+  return strtoupper($str);
 }
 
 // Write your own "tests" below. Refer to the the examples from exercise #1.
+echo "\nTesting that 'test' returns 'TEST'.\n";
+if (uppercasesAllLetters("test") == "TEST") {
+  echo "Success!";
+}
+else{
+  echo "Failed!";
+}
+echo "\n-------------------------------------------------------------------\n";
+
+echo "\nTesting that 'I' returns 'I'.\n";
+if (uppercasesAllLetters("I") == "I") {
+  echo "Success!";
+}
+else{
+  echo "Failed!";
+}
+echo "\n-------------------------------------------------------------------\n";
+
+echo "\n###################################################################\n";
+echo "Tests complete.\n";
```

Comment:
Nice work! Are there other scenarios that aren't covered in your tests? Maybe `"sTRingZ liKe THis"` `"0r th1s?"`